DigitalLearningSolutions.Web/Attributes/ValidateAllowedApplicationTypeAttribute.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace DigitalLearningSolutions.Web.Attributes
+{
+    using System;
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.Filters;
+    using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
+    public class ValidateAllowedApplicationTypeAttribute : Attribute, IActionFilter

```

Comment:
The logic is different. For one, Support is only available on tracking system and frameworks.

Reply:
@ibrahimmunir14 If the attribute constructor were to take an `IEnumerable<ApplicationType>` of applications in which the page were included, I think this could work quite well for the Support page, and any other multi-app pages we have.

Reply:
@ibrahimmunir14 Let's put this in a different ticket so we can get this merged actually.

Reply:
Created https://softwiretech.atlassian.net/browse/HEEDLS-608 to change this